src/css/application.css

```diff
@@ -566,11 +583,16 @@ body {
 /** @define notification-list */
 
 .notification-list__notification {
-  margin: 0.2rem 0;
+  font-size: var(--font-size-menu);
```

Comment:
This seems unrelated to the other stuff in here?

Reply:
Somewhat. Kinda rolling it into general "chrome/menu/etc." design tweaks to reign in the sizing and spacing of everything and make things look more standard. Would you be totally averse to keeping it in here?